ci/ci/environment.py

```diff
@@ -21,6 +21,8 @@
 
 if CLOUD == 'gcp':
     REGION = get_gcp_config().region
+    PROJECT = get_gcp_config().project
 else:
     assert CLOUD == 'azure', CLOUD
     REGION = get_azure_config().region
+    PROJECT = None
```

Comment:
nit: Since project can never be not-None in azure, I would rather the variable just not exist in that environment at all. Since `PROJECT` is basically never going to take on multiple values (and therefore is not super useful as a function parameter), can you instead access the project config variable inside the GCP-specific function in utils that uses it? Then you also don't need these asserts that it's not None.